src/app/busca-ativa/citopatologico/Cito.tsx

```
@@ -37,63 +36,54 @@ export const Cito: React.FC<CitoProps> = ({
 	const [showSnackBar, setShowSnackBar] = useState({
 		open: false,
 	});
-	const [filtrosAplicados, setFiltrosAplicados] = useState(false);
+	const [filtros_aplicados, setFiltros_aplicados] = useState(false);
```

Biome (1.9.4)

[error] This const name should be in camelCase or PascalCase.

Safe fix: Rename this symbol in camelCase.


(lint/style/useNamingConvention)